### DISCUSSION TITLE
Can ferdium be set up to resist tracking/fingerprinting?

### DISCUSSION THREAD

**Discussion by sed-i** [ferdium/ferdium-app#1311] [Q&A]
Most of the services that ferdium supports utilize tracking/fingerprinting.
Can ferdium be set up to resist those?

**Comment by vraravam** [comment]
how is this possible? Ferdium doesn't yet support any extensions. If this is possible via any extra headers to be sent on all http requests, yes that can be incorporated - but will need some development effort. We will gladly accept code contributions toward the same

**Reply by sed-i** [reply]
I am not familiar with electronjs, but I was wondering if it's possible to:
- Disable [device access](https://www.electronjs.org/docs/latest/tutorial/devices)
- Follow [security recommendation checklist](https://www.electronjs.org/docs/latest/tutorial/security#checklist-security-recommendations)
- Any other similar hardening you can think of
